pages/faq.tsx

```diff
@@ -219,7 +219,48 @@ const questions: ReadonlyArray<{
     question:
       'I get the error, "Your database needs to be upgraded (v10 vs v14)."',
     id: 'database-error',
-    answer: `You are running phase 3 on an old phase 2 database. You need to delete your datadir located at ~/.ironfish`,
+    answer: (
+      <div>
+        You are running phase 3 on an old phase 2 database. You need to delete
+        your old datadir. Please first try the command{' '}
+        <code>ironfish reset</code>. If that does not work try{' '}
```

Comment:
Until we release we should not recommend ironfish reset, then we should only recommend it